src/core/user/token/user-token.connector.ts

```diff
@@ -0,0 +1,53 @@
+import { Store } from 'redux';
+
+import { IState } from 'src/app/store/app.types';
+import { IService } from 'src/services/service.types';
+import { ITokenService } from 'src/services/token/token.types';
+
+import { selectRefreshToken } from './user-token.selectors';
+import { setUserTokenAction } from 'src/core/user/token/user-token.actions';
+import { IUserTokenState } from 'src/core/user/token/user-token.types';
+import { Omit } from 'src/utils/typescript';
+
+export type IAccessToken = Pick<IUserTokenState, 'accessToken'>;
+
+export const connectUserTokenFactory = (
```

Comment:
this needs to be called when we create the store for the first time, even if we need a singleton wrapper to cache the result of this function, like maybe app.services or something like that not sure